src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -10,7 +13,15 @@ public class AuthenticationService {
      * @return true if user by email exists and passed password is equal to user's password.
      * Return false in any other cases.
      */
+
+    private UserService userService = new UserService();
+
     public boolean login(String email, String password) {
-        return false;
+        User user = userService.findByEmail(email);
+
+        if (user == null) {
+            return false;
+        }
+        return user.getPassword().equals(password);
```

Comment:
```suggestion
        return user != null && user.getPassword().equals(password);
```
you can simplify  it 😉